src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -1412,7 +1458,7 @@ class ArchiveBuilder::CDSMapLogger : AllStatic {
   // the narrowOop version of the requested address.
   //     0x00000007ffc7e840 (0xfff8fd08) java.lang.Class
   //     0x00000007ffc000f8 (0xfff8001f) [B length: 11
-  static void print_oop_with_requested_addr_cr(outputStream* st, oop source_oop, bool print_addr = true) {
+  static void print_oop_with_requested_addr_cr(outputStream* st, oop source_oop, bool print_addr = true, bool print_aot_init = false) {
```

Comment:
Pre-existing but it seems very odd to me to have a function called `print_oop_with_requested_addr` and then have a parameter to control whether we print the addr.

Reply:
I changed the function name to `print_oop_info_cr()` and added comments about whether address(es) are printed, and in what format.